### PR TITLE
Milestone #3

### PR DESCRIPTION
**Pull request by kensayo**
### Milestone #3

- Added 'In Magazine', 'Editors Pick', 'Subscribe' and 'Newsletter Section'
- Fixed Linters
- Responsive for mobile, tablet, and desktop versions.
- Requested changes completed.
- Update Readme file

**Good day and thanks a lot**

### CODE REVIEW

**Comment by Spayco** [comment]
Hello, you can fix merge conflicts by doing `git pull origin main`, the files that have conflicts (in your case, index.html, and styles.css) would be changed and tells you where the conflict is, you can follow [this video](https://youtu.be/JtIX3HJKwfo) to see how to handle those changes, hope this helps, 



Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [ ] If you need some assistance with your project or are having a hard time understanding something. Kindly feel free to schedule a call with a full-time TSE [using this link](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-#). Or you can reach out to any TSE on slack, we shall gladly help. In future projects, you don't need to submit your project for review if you're aim is to get assistance. Only, submit your project for review, if you feel its complete and ready to be reviewed.
- [ ] Secondly, its better you don't keep closing and opening new pull requests until project completion/approval. It causes a lot of confusion for us the reviewers. except for the case of this bootstrap project where all milestones 1, 2 & 3 should all be reviewed on the same pull request, while you're completing other projects unless a project or milestone is approved, please, keep submitting the same pull request, don't create another.

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+}
+
+.mini-logo {
+  height: 30px;
+  width: 30px;
+}
+
+.mobile {
+  display: none;
+}
```
Comment (Spayco) [inline styles.css:175]: - [ ] use `d-none` Bootstrap class instead of this

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] some images aren't loading, please fix that
- [x] Article images should be links
* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [x] Sponsored Insight articles are centered in the tablet view

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by kensayo** [comment]
> # Changes Requested
> Hello Team! 👋
> Awesome job so far!! 👍 you still need to make some changes though!
> 
> ## Required Changes ♻️
> * [ ]  some images aren't loading, please fix that
> * [x]  Article images should be links
> 
> * [ ]  Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
> * [x]  Sponsored Insight articles are centered in the tablet view
> 
> ### please make the appropriate changes and ask for another code review ✔️
> Happy Coding!
> 
> Best,
> Aziz Mejri
> _contact me on [twitter](https://twitter.com/iSpaYco) or slack (@Spayco) if you need any help_

All issues fixed, I wasn't able to find issues with image loading.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1503 @@
...
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr>
+
+      <div class="NewspaperMain container-fluid">
+        <div class="header-tech-science pb-2">
+          <a class="text-danger fw-bold inMagTitle" href="#">Tech & Science</a>
```
Comment (euqueme) [inline index.html:1352]: - [ ] The heading should be surrounded by an `<h>` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1503 @@
...
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr>
+
+      <div class="NewspaperMain container-fluid">
+        <div class="header-business pb-2">
+          <a class="text-danger fw-bold inMagTitle" href="#">Business</a>
```
Comment (euqueme) [inline index.html:1299]: - [ ] The heading should be surrounded by an `<h>` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1503 @@
...
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr>
+
+      <div class="NewspaperMain container-fluid">
+        <div class="header-world pb-2">
+          <a class="text-danger fw-bold inMagTitle" href="#">World</a>
```
Comment (euqueme) [inline index.html:1244]: - [ ] The heading should be surrounded by an `<h>` tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1503 @@
...
+            <span class="ep-cap">The president-elect plans to reverse the so-called "Muslim ban" and the exit from the
+              Paris Climate Accords.</span>
+          </article>
+        </div>
+      </div>
+      <hr>
+
+      <div class="NewspaperMain container-fluid">
+        <div class="header-united pb-2">
+          <a class="text-danger fw-bold inMagTitle" href="#">U.S.</a>
```
Comment (euqueme) [inline index.html:1191]: - [ ] The heading should be surrounded by an `<h>` tag

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)
### Hello @kensayo and @craigs40 :balloon: :lollipop: 

### Thank you for working on the changes requested by the previous reviewers :medal_sports: :medal_sports: your project looks very similar to the original and there aren't missing sections, claps for the good work done so far :clap: :clap: :clap: let's work on the following changes :construction_worker: :construction_worker: 

### General

- [ ] NOT make [common mistakes](https://speckyboy.com/good-bad-css-practices) in your CSS styles
> I put comments directly in your CSS file where selectors were being repeated, create classes so you keep your CSS code DRY.

### Implementation requirements

* [x] Use Bootstrap CSS classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))

> you should use bootstrap classes and the CSS file should be as small as possible so let's use bootstrap classes for the following selectors:
- list-style: none
- height: auto
- text-decoration: none;
- color: (black, white, red, blue)
- background-color: (black, white, red, blue)
- display: (all that applies)
- justify-content: (all that applies)
- position: (all that applies)
- font-size: (al that applies)

### All Milestone requirements

#### Mobile Screens
##### Navbar
- [ ] The login, SUBSCRIBE button, and hamburger menu need to be aligned to the rightmost part of the navbar, also the Newsweek logo should be bigger and the red background (the navbar itself) should be smaller in height, also it should take the whole screen width, no white spaces should be showing left and right.
- Clone
image
- Original
image
##### Repeating sections
- [ ] Several of your sections are being repeated in this screen size there should be one of each section and the order should be respected: 
- Clone (wrong order)
TOP STORY -> FEATURED STORIES -> THE DEBATE -> OPINION -> SPONSOR INSIGHT / RANKINGS -> MORE STORIES -> MY TURN -> CULTURE & TRAVEL -> FREE SIGN UP form -> SPONSOR INSIGHT / RANKINGS  -> FEATURED STORIES -> SPONSOR INSIGHT / RANKINGS -> TOP STORY -> MY TURN -> CULTURE & TRAVEL  -> MORE STORIES -> LATEST NEWS -> THE DEBATE -> OPINION -> SPONSOR INSIGHT / RANKINGS -> FREE SIGN UP form -> SPONSOR INSIGHT / RANKINGS -> IN THE MAGAZINE -> EDITOR'S PICK -> Main articles (US, World, Bussiness, etc) -> SUBSCRIBE -> NEWSLETTER
- Original (right order)
TOP STORY -> FEATURED STORIES -> THE DEBATE -> OPINION -> SPONSOR INSIGHT / RANKINGS -> MORE STORIES -> LATEST NEWS -> MY TURN -> CULTURE & TRAVEL -> FREE SIGN UP form -> SPONSOR INSIGHT / RANKINGS -> IN THE MAGAZINE -> EDITOR'S PICK -> Main articles (US, World, Bussiness, etc) -> SUBSCRIBE -> NEWSLETTER 
##### Main news sections (U.S., World, etc.)
- [ ] The images of the small articles should take 60% of the screen width, they should be aligned with the big articles and there should be more margin separating the big articles from the small ones
- Clone
image
- Original
image
##### SUBSCRIBE
- [ ] The images, text and subscribe button should be centered and the bold text should be on top of the images
- Clone
image
- Original
image
##### NEWSLEETTER
- [ ] The image should be on top and take the whole section width and half of the section height, the text should be centered, and the email input field next to the FREE SIGN UP button
- Clone
image
- Original
image
##### Footer
- [ ] The bottom links should be aligned with the NEWSWEEK logo on the left
Clone
image
Original
image
#### Tablet Screens
##### Navbar
- [ ] Should be small in this screen size and contains only the NEWSWEEK logo at the leftmost part and the LOGIN link SUBSCRIBE button and hamburger menu at the rightmost part
- Clone
image
- Original
image
##### Main articles (US, World, Business, etc.)
- [ ] The aside articles should have less margin on the left with the main articles **Fix it in Desktop screens too**
Clone
image
Original
image
##### SUBSCRIBE 
- [ ] The SUBSCRIBE button should be aligned to the left like the upper text **Fix it also for Desktop screens**
- Clone
image
- Original
image
##### NEWSLETTER
- [ ] The email input and FREE SIGN UP button should be next to each other 
Clone
image
Original
image
#### Desktop Screens
##### Navbar
- [ ] The links should be better distributed in the space right now they look centered in the Navbar but they should look justified
- Clone
image
- Original
image
#### All Screens
##### LATEST NEWS
- [x] The Heading of this section should be written in all caps
- Clone
image
Original
image
##### SPONSORED INSIGHT/ RANKINS
- [ ] The images for these sections aren't showing and also the blue heading should be written with all caps
- Clone
image
- Original
image
##### Main articles (U.S., World, Bussiness)
- [ ] The headings for these sections should be surrounded by an `<h>` tag
##### SUBSCRIBE and NEWSLETTER
- [x] The headings for these sections should be written in all caps
Clone
image
Original
image

### That's it, team!  :raised_hands: 
### Please :pray:  do the changes :technologist: :technologist:  and submit back for a code review when you are done :memo: 
Any question? ask me here but ping me on slack **@ Maru** :woman_technologist: 
### Happy coding! :rocket: 
PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 
- Great work improving on most of the issues brought up earlier by the previous TSE!

## However, there are some changes required 👀 🧰 🔨 🔧 
- On mobile view some of your sections display stretched or distorted images. Please fix them: 
image
image

- On tablet mode some of the thumbnails on the right side here are also stretched. Please fix them as well:
image

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**